snuba/migrations/runner.py

```diff
@@ -156,12 +160,27 @@ def get_status(migration_key: MigrationKey) -> Status:
             group_migrations: List[MigrationDetails] = []
             group_loader = get_group_loader(group)
 
-            for migration_id in group_loader.get_migrations():
+            migration_ids = group_loader.get_migrations()
```

Comment:
I'd probably convert this to a `set` instead of doing it later:
```
migration_ids = set(group_leader.get_migrations())
```

Reply:
The reason I didn't do this is to maintain the original order of `get_migrations()`. A set is unordered so there is no guarantee that the order will stay the same.